### DISCUSSION TITLE
Retry button in offline menu

### DISCUSSION THREAD

**Discussion by davidmayr** [PrismLauncher/PrismLauncher#1767] [Ideas]
When the launcher asks you to put in a name for offline play (I get that if I leave the launcher open for too long) just please add a retry button. It would try to reauthenticate the profile and start with online play.

Would be very helpful!

**Comment by Ari-43** [comment]
This would also be helpful for anyone who may launch the launcher while not on a network, and then later connects.

**Comment by Trial97** [comment]
Yeah can we move this to one of the following issues: https://github.com/PrismLauncher/PrismLauncher/issues/520
or https://github.com/PrismLauncher/PrismLauncher/issues/592 or https://github.com/PrismLauncher/PrismLauncher/issues/920
This looks like a possible solution for them(the best would be to not have any user input and work on its own).
